NashTeamAlpha/CustomerIncidentPortal
# Jd gitignore fix

## Pull Request Description

Include the applicable information from the list below here 

1. Added obj and bin folders to gitignore.
1. Deleted said folders to avoid merge issues with dev branch.

## Steps to Test

Include the applicable information from the list below here

1. Switch to branch jd-gitignore-fix
2. Open folder in VS Code
3. Verify that bin and obj are located at the bottom of .gitignore 

## Link to Feature Ticket

Provide the issue number that this pull request fixes
Example: #19
